Formula/imgcat.rb

```diff
@@ -0,0 +1,18 @@
+class Imgcat < Formula
+  desc "Displays one or more images inline at their full size."
+  homepage "https://www.iterm2.com/documentation-images.html"
+  url "https://raw.githubusercontent.com/gnachman/iTerm2/master/tests/imgcat"
```

Comment:
We need a stable, tagged release for this rather than a single file on `master`.